src/libstd/thread/local.rs

```diff
@@ -106,12 +106,20 @@ pub struct LocalKey<T: 'static> {
 #[stable(feature = "rust1", since = "1.0.0")]
 #[allow_internal_unstable]
```

Comment:
OK, it shouldn't be redundant because `__thread_local_inner!` is marked unstable, but something is failing to check that!